### ISSUE TITLE
`--force-colors` not working 

### ISSUE THREAD

**Issue by felipecrs** [kubecolor/kubecolor#121]
It used to work, but with latest version it no longer does.

```console
$ k3d cluster create

$ docker run --rm --network=host \
    -v "${HOME}/.kube/config:/root/.kube/config" \
    -v "$(/bin/which kubectl):/usr/local/bin/kubectl" \
    -v "$(/bin/which kubecolor):/usr/local/bin/kubecolor" \
    ubuntu kubecolor --force-colors get sc
<prints with no colors>
```

![image](https://github.com/kubecolor/kubecolor/assets/29582865/7063564c-3d0d-41cf-b132-9b44961b3089)

**Comment by applejag** [comment]
Hi @felipecrs, thanks for reporting this!

We had the opposite problem before, where it was adding too much colors.

In the attempt to fix that in #118, it's possible we made it too colorless, so to speak.

I wonder, if you add `-it` to the docker command (so it's `docker run --rm -it ...`), does the colors work then? Not suggesting a fix. Just curious to try aim down where the but is.

**Comment by felipecrs** [comment]
Yes, adding -it the docker command colors it, but even without --force-colors, rendering --force-colors useless.

In my use case. I'm running it within a script, in a pipe, and I'm capturing the output to print out later.

**Comment by applejag** [comment]
Yeah this definitely sounds like a bug

**Comment by applejag** [comment]
Alright, found out what's going on. Kubecolor correctly outputs all the colors, but the [gookit/color](https://pkg.go.dev/github.com/gookit/color) package that we introduced in v0.3.0 does its own terminal detection.

We can just tell it to force colors there too, but we've a potential issue: we do color translation depending on what your terminal supports. If your terminal only supports 8-bit colors, then the RGB colors are translated to 8-bit.

This is currently done during parsing of the config, to make sure it only happens once. Which means it happens before we know if we have passed the `--force-colors` flag.

Should maybe be documented then that `COLORTERM` needs to be set to an appropriate value. Question is if `--force-colors` should override this?
